pyproject.toml

```diff
@@ -20,15 +20,14 @@ authors = [
 ]
 description = "Python/C++ library for distribution power system analysis"
 readme = "README.md"
-license = { text = "MPL-2.0" }
+license = "MPL-2.0"
 classifiers = [
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: C++",
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
     "Intended Audience :: Science/Research",
-    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
```

Comment:
why?

Reply:
Deprecation of setuptools. This is the new way of specifying license.